styles/globals.scss

```diff
@@ -6,6 +6,7 @@ $nhsFont: “Frutiger W01”, Arial, sans-serif;
 
 html {
   background-color: inherit;
+  font-family: "Frutiger W01", Arial, sans-serif !important;
```

Comment:
`!important ` really? 😁

Reply:
@ReubenRobertsHEE don't take away my comfort blanket! 😅
Default Amplify css has had a habit of overriding things so was taking no chances (another PR to fix 😁).
Happy to remove if you're feeling lucky?

Reply:
Definitely not, I'm all for comfort blankets and such like 😀 ⛑️